mobius-rx3/src/main/java/com/spotify/mobius/rx3/UnknownEffectException.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * -\-\-
+ * Mobius
+ * --
+ * Copyright (c) 2017-2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.mobius.rx3;
+
+import static com.spotify.mobius.internal_util.Preconditions.checkNotNull;
+
+/**
+ * Indicates that a {@link MobiusEffectRouter} has received an effect that it hasn't received
+ * configuration for. This is a programmer error.
+ */
+class UnknownEffectException extends RuntimeException {
```

Comment:
Changed to package

Reply:
The reason why this class was made public was in case someone would like to catch it and treat those errors specially. It's not likely to be a super common use case, but it's possible, I think.

Reply:
OK, running a git grep `UnknownEffectException` inside the Spotify android client, I see 0 usages. I change it to public tho